src/constants.js

```diff
@@ -213,6 +223,9 @@ const MAINTENANCE_TYPE = {
   REINDEX_TABLES: 'REINDEX TABLES',
   DAILY_DB_MAINTENANCE: 'DAILY DB MAINTENANCE',
   CLEAR_MAINTENANCE_LOGS: 'CLEAR MAINTENANCE LOGS',
+  IMPORT_SCHEDULE: 'IMPORT SCHEDULE',
```

Comment:
_nitpick_: we have the _ in values for IMPORT_STATUSES and don't use it here; can we be consistent?

Reply:
I do see an understand the concern here. I am open to changing the three new values. But the system has been in production for the other values in this "Enum". So my thought is to add the underscore for the new values only. If we need the others updated that would need to be another PR as there would need to be a migration file update as well as the values are stored in the DB for the maintenance log.

Reply:
I have made the change for the new values, Please make a PR if we need the other values updated as well.